openmdao/docs/theory_manual/setup_stack.rst

```diff
@@ -70,35 +70,31 @@ called on all subsystems, you can be sure that this information will be availabl
 
 Here is a quick guide covering what you can do in the `setup` and `configure` methods.
 
-**setup**
-
- - Add subsystems
- - Issue connections
- - Assign linear and nonlinear solvers at this group level
- - Change solver settings for any solver at this group level
- - Assign Jacobians at this group level
- - Set system execution order
- - Add desvars, objectives, and constraints
- - Add a case recorder to the group or a solver in that group.
-
-**configure**
-
- - Issue connections
- - Assign linear and nonlinear solvers to subsystems
- - Change solver settings in subsystems
- - Assign Jacobians to subsystems
- - Set execution order in subsystems
- - Add a case recorder to a subsystem or a solver in a subsystem.
-
-**Things you should never do in configure**
-
- - Add subsystems
- - Delete subsystems
+==================================================================================== ======= ===========
+Action                                                                               setup   configure
+==================================================================================== ======= ===========
+Add subsystems                                                                          o
+Delete subsystems                                                                       o
+Issue connections                                                                       o        o
+Set system execution order                                                              o        o
```

Comment:
Should we split this out into 'at this group level' and 'in subsystems' like you did with the others?